ivy/array/general.py

```diff
@@ -1078,5 +1078,22 @@ def get_num_dims(self: ivy.Array, /, *, as_array: bool = False) -> int:
         ret
             Shape of the array
 
+        Examples
```

Comment:
you should fix the failing docstring tests for these examples

Reply:
fixed